weaviate/helm/weaviate/values.yaml

```diff
@@ -0,0 +1,39 @@
+# # Default values for weaviate.
+# # This is a YAML-formatted file.
+# # Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+ingress:
+  enabled: true
+  className: "nginx"
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+  service:
+    port: 80
+
+weaviate:
+  image:
+    registry: dkr.plural.sh
+    tag: 1.18.0
+    repo: weaviate/semitechnologies/weaviate
```

Comment:
@michaeljguarino can you confirm that all images in `vendor_images.yaml` will be sync within `dkr.plural.sh` registry with application name as prefix?

Reply:
that's correct